nemo_curator/modules/fuzzy_dedup/connectedcomponents.py

```diff
@@ -58,7 +58,7 @@ def __init__(
         else:
             self._logger = logger
 
-    def cc_workflow(self, output_path):
+    def __call__(self, output_path):
```

Comment:
The main reason this was not kept as the standard `__call__` method is because those typically have an assumption of `DocDataset` in `DocDataset` out. Since this doesn't follow the same pattern, it cannot be chained well in `Sequential`. Not sure if we should revert back to the old name or expand the definition of what this method can do.